src/controllers/words.js

```diff
@@ -199,3 +199,22 @@ export const putWord = (req, res) => {
       return res.send({ error: 'An error has occurred while updating the word, double check your provided data.' });
     });
 };
+
+export const deleteWord = (req, res) => {
+  const { params: { id } } = req;
```

Comment:
There's a chance that a `WordSuggestion` and `GenericWord` document might reference to a `Word` document with either the `originalWordId` or even the `merged` key. I think we should also make a pass through those documents and remove references from those objects.